### PR TITLE
Publish 2020-03-Aproductsuverystory

### PR DESCRIPTION
**Pull request by m00nlitcat**

### CODE REVIEW

**Comment by m00nlitcat** [comment]
@matthcap & @kacper-r here's the blog post for review, hoping everything works :)

File: _posts/2020-03-31-a-product-survey-story.markdown [hunk 1]
```diff
@@ -0,0 +1,87 @@
+
+--->
```
Comment (matthcap) [inline _posts/2020-03-31-a-product-survey-story.markdown]: we can delete this

File: _posts/2020-03-31-a-product-survey-story.markdown [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+---
+layout: post
+title: "A Product Survey Story: What we learned"
```
Comment (matthcap) [inline _posts/2020-03-31-a-product-survey-story.markdown]: all caps for the title

File: _posts/2020-03-31-a-product-survey-story.markdown [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+date: 2020-04-02 10:37:03 -0600
+comments: true
+published: true
+categories: ["post"]
+tags: ["Community news", "Development", "NuGet", "Stories", "Feature", "Python"]
+author: "Cat Richardson"
+---
+
+# A Product Survey Story: What we learned 🧐
```
Comment (matthcap) [inline _posts/2020-03-31-a-product-survey-story.markdown]: I think we want all caps for post header

File: _posts/2020-03-31-a-product-survey-story.markdown [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+Thanks to your candid participation, we learned a tremendous amount about what you value most about MyGet and what we need to change. We are eager to share the results with you too, to help us all better understand the community that makes up MyGet! Here are our top takeaways from the results.
+
+
+## Built by developers, for developers
+
+First off, we learned more about you! As we think about continuing to build out the functionality of MyGet, it's so important to understand who actually uses our platform and what challenges you face. (In a world that is growing more and more remote, it’s important to stay connected as a community!)
+
+Of the more than 750 folks that participated, more than 85% of you self-identified as having a technical role on your team, whether that be as a developer/software engineer (58%), architect (12%), DevOps/IT engineer (12%) or a Tester/QA team member (3%).
+
+![Who uses my get?](images/Who-uses-MyGet?.jpg )
```
Comment (matthcap) [inline _posts/2020-03-31-a-product-survey-story.markdown]: I think this should be `/images/2020/who-uses-MyGet?.jpg`

File: _posts/2020-03-31-a-product-survey-story.markdown [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+
+**.NET, JavaScript HTML: The Top 3 \
+**In its early days, MyGet was lovingly introduced as  “NuGet-as-a-service”. That was the need it was built to solve—an easy way for you to get the benefits of a private NuGet server without the headache of managing installation or infrastructure. So it’s not surprising to see Microsoft .NET languages used most by MyGet community members.
```
Comment (matthcap) [inline _posts/2020-03-31-a-product-survey-story.markdown]: the header line should. be
```
## .NET, JavaScript, HTML: The Top 3
```

File: _posts/2020-03-31-a-product-survey-story.markdown [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+But on the other hand, our Python PyPI support is one of the most recent functionalities we have added to MyGet. It’s reassuring to see how many members of the MyGet community benefitted from the addition of Python packages alongside their NuGet, npm, Bower, Maven, and Composer packages. In less than six months, Python PyPI packages climbed to the number 5 spot in our list of most-used package formats, surpassing other package manager types that we’ve supported for much longer and knocking on Maven’s doorstep for the number 4 spot.
+
+
+![What package manager do you use?](images/What-Package-manager-does-your-team-use.jpg)
+
+
+
+## How does MyGet fit into your DevOps lifecyle?  
+
+Over 98% of you responded that you are currently using source control for the proprietary code you write, which is a good thing: source control is a must-have for modern software development. Almost 100% of you who use source control are using Git, with Github, Azure DevOps (split between cloud and server), Bitbucket, GitLab, and Assembla coming in as your top five hosting platforms. \
```
Comment (matthcap) [inline _posts/2020-03-31-a-product-survey-story.markdown]: delete `\` at the end of this line

File: _posts/2020-03-31-a-product-survey-story.markdown [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+
+
+![What tools do you manage your source code with?](images/Whats-source-code-do-you-use.jpg)
+
+
+One of the most effective ways to leverage the artifacts you store in MyGet is to integrate MyGet with your CI/CD processes. More than 85% of you responded that you have already embraced the continuous delivery and integration, using a myriad of tools such as Azure DevOps, Jenkins, TeamCity, GitLab, Bamboo, Appveyor, and Travis CI to run your builds and push build artifacts to MyGet.
+
+
+![alt_text](images/What-do-you-use-CI-CD.jpg)
```
Comment (matthcap) [inline _posts/2020-03-31-a-product-survey-story.markdown]: replace `alt_text`

File: _posts/2020-03-31-a-product-survey-story.markdown [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+
+
+
+## Why do you use MyGet?
+
+Because of its nature as a hub between multiple spokes of the SDLC, the number of different use cases for MyGet can be quite varied. That said, over 68% you reported using MyGet to share your private packages within your internal team. The next three most-reported use cases each had about the same number of responses:  23% of you use MyGet to store build artifacts, 21% responded that you use MyGet to control the open source packages used in your dev environment, followed closely by the 20% of you using MyGet to host open-source packages you have built and shared with the community/public. 
+
+
+![alt_text](images/What-is-your-primary-use.jpg)
```
Comment (matthcap) [inline _posts/2020-03-31-a-product-survey-story.markdown]: `alt_text`

File: _posts/2020-03-31-a-product-survey-story.markdown [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+
+
+## What did you ask us to prioritize next?
+
+ \
```
Comment (matthcap) [inline _posts/2020-03-31-a-product-survey-story.markdown]: delete `\`

File: _posts/2020-03-31-a-product-survey-story.markdown [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+MyGet’s success depends on our ability to continually improve features and build new ones that make your lives easier and help you ship better products. Despite the rise and adoption of 3rd-party CI/CD platforms across the MyGet Community, more than 75% of you said that updating MyGet’s build services to support .NET Core 3 and C# 8 was still important to you. Improving uptime and availability, your second-most requested improvement, has been high on our priority list for a while as well, so we are glad to see that this aligns with the concerns you have. Your feedback has been enormously helpful as we’ve been culling down our roadmap for this next fiscal year. Thank you again for your candid input—we are going to be able to be much more focused on delivering the improvements that matter to you because of it!
```
Comment (matthcap) [inline _posts/2020-03-31-a-product-survey-story.markdown]: grammar edit I just caught:
`MyGet's success depends on our ability to continually improve and build new features that make your...`

File: _posts/2020-03-31-a-product-survey-story.markdown [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+
+![What should we do next?](images/What-should-we-do-next.jpg)
+
+
+
+## Wrapping things up!
+
+We’re glad we could share these insights with you. The more we can connect and learn how you like using MyGet, the better we can help you manage your packages and ship better products. 
+
+Please reach out! We love hearing your stories and suggestions. You can reach us at [support@myget.org,](mailto:support@myget.org) start a chat with us at [https://myget.org,](https://myget.org) or connect with us on [Twitter](https://twitter.com/mygetteam) or [Facebook](https://facebook.com/mygetteam). And be sure to regularly check out our [UserVoice](https://myget.uservoice.com/forums/135675-general) **** page to vote on our roadmap and stay up to date with our most recent releases (like our [NuGet Symbol Server update](http://blog.myget.org/post/2020/03/05/myget-nuget-symbols-snupkgs.html), which was your top-voted request on UserVoice)!
```
Comment (matthcap) [inline _posts/2020-03-31-a-product-survey-story.markdown]: delete `****` after UserVoice link

**Review by matthcap — CHANGES_REQUESTED** [review]
great draft, looks like there are a few minor edits to make and we need to update the image paths to `/images/2020/filename.jpg` instead of `images/filename.jpg`, then we should be good to have Kacper review!

File: _posts/2020-03-31-a-product-survey-story.markdown [hunk 12]
```diff
@@ -6,6 +7,7 @@ published: true
 categories: ["post"]
 tags: ["Community news", "Development", "NuGet", "Stories", "Feature", "Python"]
 author: "Cat Richardson"
+----
```
Comment (matthcap) [inline _posts/2020-03-31-a-product-survey-story.markdown]: this should be just three `---`, I believe @m00nlitcat

File: _posts/2020-03-31-a-product-survey-story.markdown (hunk 5 above)
Comment (matthcap) [reply _posts/2020-03-31-a-product-survey-story.markdown]: still missing a `,` between JavaScript and HTML I think!

File: _posts/2020-03-31-a-product-survey-story.markdown [hunk 13]
```diff
@@ -0,0 +1,83 @@
...
+
+First off, we learned more about you! As we think about continuing to build out the functionality of MyGet, it's so important to understand who actually uses our platform and what challenges you face. (In a world that is growing more and more remote, it’s important to stay connected as a community!)
+
+Of the more than 750 folks that participated, more than 85% of you self-identified as having a technical role on your team, whether that be as a developer/software engineer (58%), architect (12%), DevOps/IT engineer (12%) or a Tester/QA team member (3%).
+
+![Who uses my get?](images/2020/Who-uses-MyGet?.jpg)
+
+
+## .NET, JavaScript HTML: The Top 3
+**In its early days, MyGet was lovingly introduced as  “NuGet-as-a-service”. That was the need it was built to solve—an easy way for you to get the benefits of a private NuGet server without the headache of managing installation or infrastructure. So it’s not surprising to see Microsoft .NET languages used most by MyGet community members.
```
Comment (matthcap) [inline _posts/2020-03-31-a-product-survey-story.markdown]: still need to delete the two `**` at the start of "In its early days" too!

File: _posts/2020-03-31-a-product-survey-story.markdown [hunk 14]
```diff
@@ -0,0 +1,83 @@
...
+Of the more than 750 folks that participated, more than 85% of you self-identified as having a technical role on your team, whether that be as a developer/software engineer (58%), architect (12%), DevOps/IT engineer (12%) or a Tester/QA team member (3%).
+
+![Who uses my get?](/images/2020/Who-uses-MyGet?.jpg "Who-uses-MyGet?")
+
+
+## .NET, JavaScript HTML: The Top 3
+**In its early days, MyGet was lovingly introduced as  “NuGet-as-a-service”. That was the need it was built to solve—an easy way for you to get the benefits of a private NuGet server without the headache of managing installation or infrastructure. So it’s not surprising to see Microsoft .NET languages used most by MyGet community members.
+
+
+[What programming language is most widely used?](/images/2020/What-programming-language-used.jpg "What-programming-language-used")
```
Comment (matthcap) [inline _posts/2020-03-31-a-product-survey-story.markdown]: this one is missing the `!` in front of the `[What programming language]`

**Comment by m00nlitcat** [comment]
@matthcap it all works now, thanks for the help :)

**Comment by m00nlitcat** [comment]
@matthcap updated more!

File: _posts/2020-03-31-a-product-survey-story.markdown [hunk 15]
```diff
@@ -0,0 +1,83 @@
...
+title: "A Product Survey Story: What We learned"
+date: 2020-04-02 10:37:03 -0600
+comments: true
+published: true
+categories: ["post"]
+tags: ["Community news", "Development", "NuGet", "Stories", "Feature", "Python"]
+author: "Cat Richardson"
+---
+
+# A Product Survey Story: What We learned 🧐
```
Comment (matthcap) [inline _posts/2020-03-31-a-product-survey-story.markdown]: @m00nlitcat we should capitalize the "l" too!

**Comment by m00nlitcat** [comment]
@matthcap I've updated the blog :)

**Review by matthcap — APPROVED** [review]
updates look great @m00nlitcat

**Review by kacper-r — APPROVED** [review]
looks good to me

**Pull request merged**